README.md

```diff
@@ -78,10 +78,6 @@ For pre-container setup, account prerequisites, and UI-based support, see our ar
             uptimecom/uptime-private-location:X.Y
 
```

Comment:
Please pull latest code from latest branch, since we updated the image to :uptimecom/uptime-private-location:latest